app/src/newdocs/content/products.mdx

```diff
@@ -14,6 +34,10 @@ import docsStyles from '$newdocs/components/DocsLayout/docsLayout.pcss'
 ## Work with products in Core
 --content--
 
+Clicking the “Purchase” button initiates the purchase dialog which guides you through the contract transaction process. After the transaction has completed, you will have access to the streams.
```

Comment:
I put this here because it's not really intro but didn't want to throw it away either...

Reply:
I suggest it should go above in the paid products section.

Reply:
Added the last paragraph under "Publishing your product" heading since the first paragraph was already used in paid section.